addon/templates/components/frost-multi-select.hbs

```diff
@@ -17,7 +17,7 @@
     </div>
     <div class='multi-status'>
       <span class='number-selected'>{{selected.length}} selected</span>
-      <a href='#' class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
+      <a class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
```

Comment:
@job13er @EWhite613 no objection to completely removing it, although it was behaving strange for me without it... can't remember what it was.  Another approach would be to just not use an `<a>` here at all... a `<span>` with some styling would work as well